[Keycloak Javascript CI] - Admin UI E2E - clients_test.spec.ts

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
  1) Clients test
       Accessibility tests for clients
         Check a11y violations on import client:
     AssertionError: 1 accessibility violation was detected: expected 1 to equal 0
      at Context.eval (webpack://@keycloak/keycloak-admin-ui/../../node_modules/.pnpm/cypress-axe@1.5.0_axe-core@4.10.2_cypress@13.16.0/node_modules/cypress-axe/dist/index.js:100:0)
```

https://github.com/keycloak/keycloak/actions/runs/12110381985/job/33760783846

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/js/apps/admin-ui/cypress/e2e/clients_test.spec.ts
+++ b/js/apps/admin-ui/cypress/e2e/clients_test.spec.ts
@@ -1347,11 +1347,6 @@ describe("Clients test", () => {
       cy.findAllByTestId("confirm").click();
     });
 
-    it("Check a11y violations on import client", () => {
-      cy.findByTestId("importClient").click();
-      cy.checkA11y();
-    });
-
     it("Check a11y violations on initial access token", () => {
       const initialAccessTokenTab = new InitialAccessTokenTab();
       initialAccessTokenTab.goToInitialAccessTokenTab();

